python/python/glide/async_commands/redis_modules/search/optional_params.py

```diff
@@ -0,0 +1,60 @@
+from enum import Enum
+
+
+class FieldFlags(Enum):
+    USE_FIELDS = 0
+    """
+    Store attribute bits for each term.
+    """
+    NO_FIELDS = 1
+    """
+    Does not store attribute bits for each term. Saves memory but does not allow filtering by specific attributes.
```

Comment:
minor: The description doesn't match to other enums - please choose "Store" or "Stores" ("saves", "avoids"...) and use it throughout the options